.prettierignore

```diff
@@ -0,0 +1,2 @@
+dist
```

Comment:
```suggestion
/dist
```

Reply:
This approach works better because it won't do search for the whole tree and will check only root level of the project